docs/monoDrive_home/Simulator-Configuration.md

```diff
@@ -2,148 +2,173 @@
  
 ## Simulator Configuration
 
-```
+``` json
```

Comment:
We lost the `"traffic_configuration":` section this is used. Same for the tag descriptions we need to put them back